_includes/pbc/all/install-features/202304.0/install-the-agent-assist-feature.md

```diff
@@ -0,0 +1,574 @@
+| UserAgentTableConfigExpanderPlugin | 1. In the Back Office, go to **Users > Users**. <br>2. Ensure that the **Agent** column.exists.                                                                                                                                                                                                                                                                                                  |
+| UserAgentTableDataExpanderPlugin   | 1. In the Back Office, go to **Users > Users**. <br>2. Next to a non-agent user, select **Edit**. <br>3. Select the **This user is an agent** checkbox. <br>4. Select **Update**. This takes you to the *Users List* page with the message about the successful update displayed. <br>5. Ensure that in the **Agent** column, next to the user you've updated, the **Agent** tag is displayed. |
+
+{% endinfo_block %}
+
+## Install feature frontend
+
+Follow the steps below to install the feature frontend.
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME         | VERSION          |
+|--------------|------------------|
+| Spryker Core | {{site.version}} |
+
+### 1) Install the required modules using Composer
+
+```bash
+composer require spryker-feature/agent-assist:"{{site.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the following modules have been installed:
+
+| MODULE                 | EXPECTED DIRECTORY                           |
+|------------------------|----------------------------------------------|
+| AgentPage              | vendor/spryker-shop/agent-page               |
+| AgentWidget            | vendor/spryker-shop/agent-widget             |
+| SessionAgentValidation | vendor/spryker-shop/session-agent-validation |
+
+{% endinfo_block %}
+
+### 2) Set up configuration
+
+By default, in Spryker, posting a login form (where SecurityBlocker makes its check and blocks agents who made too many failed login attempts) is locale-independent. So, to see error messages translated into different languages, you need to configure the locale to be added to the agent login path. You can do this by modifying the following configs:
+
+**src/Pyz/Yves/AgentPage/AgentPageConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\AgentPage;
+
+use SprykerShop\Yves\AgentPage\AgentPageConfig as SprykerAgentPageConfig;
+
+class AgentPageConfig extends SprykerAgentPageConfig
+{
+    /**
+     * @return bool
+     */
+    public function isLocaleInLoginCheckPath(): bool
+    {
+        return true;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that when the login form for the agent is submitted, the URL it uses contains a locale code. For example, `/de/agent/login_check` is the default value for the agent.
+
+{% endinfo_block %}
+
+Make sure to use environment variables in `config-default.php`:
```

Comment:
use the same wording for similar cases

for core feature you have `Add environment configuration for customer security:` and here you have `Make sure to use environment variables in config-default.php:`

Reply:
Adjusted everywhere and made it almost the same, MP core. MP core has different structure